src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -236,7 +236,8 @@ static UCS_CLASS_INIT_FUNC(uct_rc_mlx5_iface_t, uct_md_h md, uct_worker_h worker
 
     UCS_CLASS_CALL_SUPER_INIT(uct_rc_iface_t, &uct_rc_mlx5_iface_ops, md, worker,
                               params, &config->super, 0,
-                              sizeof(uct_rc_fc_request_t), IBV_EXP_TM_CAP_RC);
+                              sizeof(uct_rc_fc_request_t), IBV_EXP_TM_CAP_RC,
+                              UCT_IB_MLX5_RES_DOMAIN_KEY);
```

Comment:
Is it ok that all accelerated transports use the same domain key?

Reply:
yes because they share the BF register structure as well (with the same key)